tasks/frontEnd/package.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Any reason for not using linters?

Reply:
Only due to the small size of the app and the time available. Have used linters previously and would continue to do so in real world scenarios